unfoldingWord/checking-tool-wrapper
# Fixed unnecessary invalid selections alert

## Testing

- In tc checkout feature-mannycolon-6656
- Run `rm -rf node_modules && npm i`
- npm start
- Import a project that has no alignments and no checks
- Open the project in any tool
- Make an edit to a verse
- Note that the "Some selections are no longer valid..." message is NO LONGER displayed when the project has no selections.



This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword/checking-tool-wrapper/93)
